source/partials/next_release/_release.md.erb

```diff
@@ -10,29 +13,51 @@
 
 <h4>API Improvements</h4>
 
+* <%= new_api_message '19.8.0', 6570, 1, 'Mail Server config', 'mailserver-config' %>
+* <%= new_api_message '19.8.0', 6760, 1, 'Cancel Stage', 'cancel-stage' %>
+* <%= new_api_message '19.8.0', 6766, 9, 'Pipeline Config', 'pipeline-config' %>
+* <%= link_to_issue 6784, 'Introduced Delete Pluggable SCM API' %>
```

Comment:
@maheshp -- have we made a decision whether a verb API should be added in the existing API or the API version needs to be bumped up?

Depending upon the decision we make, we will need to mention it appropriately in the release notes.

\cc: @kritika-singh3

Reply:
I think it should not be added, we should bump up the version.

Reply:
The API changelog has been moved to api.gocd.org. A link to the same has been added in the release notes.